app/src/components/Navbar.js

```diff
@@ -0,0 +1,144 @@
+import React, { Fragment } from 'react'
+import { Disclosure, Menu, Transition } from '@headlessui/react'
+import { Bars3Icon, XMarkIcon } from '@heroicons/react/24/outline'
+import { PlusIcon } from '@heroicons/react/20/solid'
+
+const navigation = [
```

Comment:
Maybe we can initialize these as react states. 

One advantage is that we can provide control flow by setting the state depending on the type of user that is logged in.

Also change name to be something more intuitive, that can better differentiate between the const below.

Reply:
done in commit e1a92a9